flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/systems.cfg

```diff
@@ -73,32 +73,19 @@ lightdef.19 = Type:10#Index:2#LocalPosition:34.8,1.0,3.7#LocalRotation:270,500,0
 
-; Captain left glareshield light
-lightdef.21 = Type:12#Index:3#LocalPosition:38.97,-2.550,5.87#LocalRotation:-10,0,0#EffectFile:LIGHT_A32NX_Glareshield#PotentiometerIndex:83
-
-; Captain right glareshield light
-lightdef.22 = Type:12#Index:3#LocalPosition:38.97,-0.6,6.2#LocalRotation:-10,0,0#EffectFile:LIGHT_A32NX_Glareshield#PotentiometerIndex:83
-
-; F/O left glareshield light
-lightdef.23 = Type:12#Index:4#LocalPosition:38.97,0.7,6.2#LocalRotation:-10,0,0#EffectFile:LIGHT_A32NX_Glareshield#PotentiometerIndex:83
-
-; F/O right glareshield light
-lightdef.24 = Type:12#Index:4#LocalPosition:38.97,2.55,5.87#LocalRotation:-10,0,0#EffectFile:LIGHT_A32NX_Glareshield#PotentiometerIndex:83
-
-; Overall glareshield (fake) ambient light
-lightdef.25 = Type:12#Index:3#LocalPosition:36.5,0,6.5#LocalRotation:0,0,0#EffectFile:LIGHT_A32NX_CockpitMinimalAmbiant#PotentiometerIndex:83
-
-; Captain left glareshield (fake) ambient light
-lightdef.26 = Type:12#Index:3#LocalPosition:37.97,-2.45,4.87#LocalRotation:0,0,0#EffectFile:LIGHT_A32NX_CockpitMinimalAmbiantLow#PotentiometerIndex:83
-
-; Captain right glareshield (fake) ambient light
-lightdef.27 = Type:12#Index:3#LocalPosition:38,-0.6,4.8#LocalRotation:0,0,0#EffectFile:LIGHT_A32NX_CockpitMinimalAmbiantLow#PotentiometerIndex:83
-
-; F/O left glareshield (fake) ambient light
-lightdef.28 = Type:12#Index:4#LocalPosition:38,0.7,4.8#LocalRotation:0,0,0#EffectFile:LIGHT_A32NX_CockpitMinimalAmbiantLow#PotentiometerIndex:83
-
-; F/O right glareshield (fake) ambient light
-lightdef.29 = Type:12#Index:4#LocalPosition:37.97,2.45,4.87#LocalRotation:0,0,0#EffectFile:LIGHT_A32NX_CockpitMinimalAmbiantLow#PotentiometerIndex:83
+; Main panel flood lights
+lightdef.21 = Type:12#Index:3#LocalPosition:38.97,-2.6,5.87#LocalRotation:-10,0,-8#EffectFile:LIGHT_A32NX_MainPanelFloodEnd#PotentiometerIndex:83   ; CPT L
+lightdef.22 = Type:12#Index:3#LocalPosition:38.97,-0.6,6.2#LocalRotation:-10,0,0#EffectFile:LIGHT_A32NX_MainPanelFloodCenter#PotentiometerIndex:83  ; CPT R
+lightdef.23 = Type:12#Index:4#LocalPosition:38.97,0.7,6.2#LocalRotation:-10,0,0#EffectFile:LIGHT_A32NX_MainPanelFloodCenter#PotentiometerIndex:83   ; F/O L
+lightdef.24 = Type:12#Index:4#LocalPosition:38.97,2.6,5.87#LocalRotation:-10,0,8#EffectFile:LIGHT_A32NX_MainPanelFloodEnd#PotentiometerIndex:83     ; F/O R
+
+; Main panel flood lights ambient fx
+lightdef.25 = Type:12#Index:3#LocalPosition:38.8,-2.58,5.78#LocalRotation:0,0,0#EffectFile:LIGHT_A32NX_MainPanelFloodAmbientEnd#PotentiometerIndex:83   ; CPT
+lightdef.26 = Type:12#Index:3#LocalPosition:38.75,0,5.8#LocalRotation:0,0,0#EffectFile:LIGHT_A32NX_MainPanelFloodAmbientCenter#PotentiometerIndex:83    ; Center
+lightdef.27 = Type:12#Index:4#LocalPosition:38.8,2.58,5.78#LocalRotation:0,0,0#EffectFile:LIGHT_A32NX_MainPanelFloodAmbientEnd#PotentiometerIndex:83    ; F/O
+
+; lightdef.28 unused
+; lightdef.29 unused
```

Comment:
Last time I checked (at least this is true for circuits), skipping indexes doesn't work as anything after the skipped index basically isn't working.

Reply:
Hi @davidwalschots 

I have not encountered any issues with this. This indexing, at least when it comes to lights, _seems_ just to be an internal one as sim re-arranges these in the simulation anyway.

In attached image you can se that both `lightdef.28` and `lightdef.29` gets populated with the next definitions in `systems.cfg`

I can't find any documentation from MS or P3D that states that lightdefinitions needs to be in a particial order.

I could re-arrange them to be in sequence, but it makes little sense?

![lightdefN](https://user-images.githubusercontent.com/433429/122799824-7f876480-d2c2-11eb-8778-a4b9f917cb63.png)